data/json/items/comestibles/mutagen.json

```diff
@@ -21,182 +21,83 @@
     },
     {
         "id" : "mutagen_beast",
+        "copy-from" : "mutagen",
```

Comment:
You need to define an `abstract` type named `mutagen` at the start of this file for this to work